api/api/src/main/scala/uk/ac/wellcome/platform/api/services/ImagesRequestBuilder.scala

```diff
@@ -36,9 +37,12 @@ class ImagesRequestBuilder(queryConfig: QueryConfig)
             ImagesMultiMatcher(q.query)
           }
           .getOrElse(boolQuery)
-          .filter(queryOptions.filters.collect {
-            case filter: ImageFilter => buildImageFilterQuery(filter)
-          })
+          .must(
+            buildImageFilterQuery(queryOptions.scoredFilters)
+          )
+          .filter(
+            buildImageFilterQuery(queryOptions.unscoredFilters)
+          )
```

Comment:
Again feels strange as these are - could we not have `DocumentFilters` and `DocumentQueries`?